crates/astria-sequencer/src/app/mod.rs

```diff
@@ -562,6 +563,8 @@ impl App {
                         // removed from the mempool in `update_mempool_after_finalization`.
                         txs_to_readd_to_mempool.push((enqueued_tx, priority));
                     } else {
+                        failed_tx_count = failed_tx_count.saturating_add(1);
```

Comment:
I'm not really sure what this log is for. Do we use it for alters somewhere?

Reply:
pretty sure it's only used for logging at the moment, but useful to have